pegasus/sites.v3/code.org/views/video_player.js.haml

```diff
@@ -1,13 +1,12 @@
-%script{src: "/js/videojs-ie8.min.js"}
-%script{src: "/js/video.min.js"}
-
 %link{href: "/css/video-js.css", rel: "stylesheet"}
 
 :javascript
 
   $(window).load(function() {
     if (window.location.search.indexOf("force_youtube_fallback") !== -1)
     {
+      $.getScript("/js/videojs-ie8.min.js");
+      $.getScript("/js/video.min.js");
```

Comment:
2. I have a concern that [`$.getScript`](https://api.jquery.com/jquery.getscript/) is not a synchronous operation, so with this change there's an added possibility that the `video` tag will be added to the DOM before the videojs script is fully loaded/executed. This might not be a problem (it may be the case that videojs will detect existing `video` tags when it's first loaded and initialize them as normal), but this a new type of behavior that we should explicitly test to make sure it works as expected.

    The easiest way I can think of to manually test this locally would be to insert a delay (timeout of x seconds) before adding this script to the DOM, and see if it still behaves correctly, e.g.:

```js
setTimeout(function(){ $.getScript("/js/video.min.js"); }, 10000);
```